salt_lsp/__main__.py

```diff
@@ -60,7 +64,7 @@ def main():
 
     log_level = loglevel_from_str(args.log_level[0])
     logging.basicConfig(
-        filename="salt-server.log",
+        filename=args.log_file,
```

Comment:
Shouldn't this be?
```suggestion
        filename=args.log_file[0],
```

Reply:
The `[0]` would be needed if `nargs=1` is used. (as mentioned in the other comment)


Oh, btw, I am doing something with this that might be a little confusing. If `--log-file` isn't passed in as an arg, then `args.log_file` defaults to `None`, and that's still passed to `basicConfig()`.  However, `basicConfig` sets filename to `None` by default if it's not specified, which is why this works fine.

The related `basicConfig()` code is here:
https://github.com/python/cpython/blob/main/Lib/logging/__init__.py#L2118-L2129